docs/core/install/linux-package-manager-fedora31.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Install .NET Core on Fedora 31 - package manager - .NET Core
+description: Use a package manager to install .NET Core SDK and runtime on Fedora 31.
+author: thraka
+ms.author: adegeo
+ms.date: 12/17/2019
+---
+
+# Fedora 31 Package Manager - Install .NET Core
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
+
+This article describes how to use a package manager to install .NET Core on Fedora 31. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.
+
+## Register Microsoft key and feed
+
+Before installing .NET, you'll need to:
+
+- Register the Microsoft key
+- register the product repository
```

Comment:
Capitalize the "R" to be consistent with the other list items. And added a period.

```suggestion
- Register the product repository.
```